modules/RTC/TPCUtils.js

```diff
@@ -396,6 +613,17 @@ export class TPCUtils {
             return null;
         }
 
+        // SVC mode for VP9 and AV1 codecs.
+        if (this._isRunningInFullSvcMode(codec)) {
+            const activeEncoding = parameters.encodings[0];
+
+            if (activeEncoding.active) {
+                return height / activeEncoding.scaleResolutionDownBy;
+            }
+
+            return null;
```

Comment:
Why not make this 0?

Reply:
Actually, this method returns null for cases when the send resolution cannot be determined. I think it makes sense to return null here.